internal/provider/provider.go

```diff
@@ -782,9 +782,10 @@ func Provider() *schema.Provider {
 			"aws_apprunner_custom_domain_association":          apprunner.ResourceCustomDomainAssociation(),
 			"aws_apprunner_service":                            apprunner.ResourceService(),
 
-			"aws_appstream_fleet":         appstream.ResourceFleet(),
-			"aws_appstream_image_builder": appstream.ResourceImageBuilder(),
-			"aws_appstream_stack":         appstream.ResourceStack(),
+			"aws_appstream_fleet":                   appstream.ResourceFleet(),
+			"aws_appstream_image_builder":           appstream.ResourceImageBuilder(),
+			"aws_appstream_stack":                   appstream.ResourceStack(),
+			"aws_appstream_stack_fleet_association": appstream.ResourceStackFleetAssociation(),
```

Comment:
Because this associates a stack with with a fleet, this should be renamed to `aws_appstream_fleet_stack_association`. The files and resource functions should be renamed to match.